modin/engines/base/io/text/csv_glob_dispatcher.py

```diff
@@ -184,7 +184,7 @@ def _read(cls, filepath_or_buffer, **kwargs):
             # Max number of partitions available
             num_partitions = NPartitions.get()
             # This is the number of splits for the columns
-            num_splits = min(len(column_names), num_partitions)
+            num_splits = min(len(column_names) or 1, num_partitions)
```

Comment:
Is it possible to move this code into a separate function so that this code can be reused?

Reply:
Yes, this and other related code was moved to the `_define_metadata` function.